server/src/main/java/com/testsigma/service/TestsigmaLabDriverSettingsService.java

```diff
@@ -145,4 +143,16 @@ public void setWebCapabilities(TestDevice testDevice,
   public Integrations getLabDetails() throws IntegrationNotFoundException {
     return this.integrationsService.findByApplication(Integration.TestsigmaLab);
   }
+
+  public String getExecutionName(TestPlanResult testPlanDetails) {
```

Comment:
@vikramvr20 can we exclude runId so that we don't need to pollute methods with TestPlanResult as an argument and as its for debugging only I feel its not worth

Reply:
Addressed